variaveis compostas/17.py

```diff
@@ -1,4 +1,5 @@
 #https://www.beecrowd.com.br/repository/UOJ_2227.html
+# so pra commitar
 A,V= [int(x) for x in input().split()] #aeroportos, voos.
```

Comment:
usa nome de variáveis melhor, o que é `x`? O que é A? O que é V?

```suggestion
aeroportos,voos= [int(user_input) for user_input in input().split()] #aeroportos, voos.
```

Reply:
Resolvido :)